ocaml/gencert/test_data/certificates/pkey_rsa_8192.pem

```diff
@@ -0,0 +1,100 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Various security tools may winge at having private keys inside repositories. It is only test data in this case, could these be generated at runtime instead in the future? E.g. with openssl.

Reply:
We could generate them, I elected not to do that as it introduces more complexity in the test code.

It will be hard to get rid of this error if the tools are grepping for this string, it's needed to generate one of the invalid private keys.